src/hooks/useCarbonInit.ts

```diff
@@ -36,6 +36,17 @@ const getTokenDecimalMap = () => {
   );
 };
 
+const removeOldV1Keys = () => {
+  Object.keys(localStorage).forEach((key) => {
+    if (
+      key.match(/carbon-v1(\.1)?-([a-zA-Z0-9]+-?)+/) &&
```

Comment:
why not simply keys that start with it? What's the reason for the extra checks?

Reply:
I moved the network in the prefix to the order of [APP_NAME, NETWORK, APP_VERSION] in a previous commit, exactly to simplify this check but forgot to change it. Done.

Reply:
Added key migration and removal logic